tests/node_converters/mod_test.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import onnx
+import pytest
+
+from tests.utils.common import check_onnx_model
+from tests.utils.common import make_model_from_nodes
+
+
+@pytest.mark.parametrize('fmod_type', [0, 1])
```

Comment:
```suggestion
@pytest.mark.parametrize(
    'dividend',
    [
        [-4, 7, 5, 4, -7, 8],
        [-4.3, 7.2, 5.0, 4.3, -7.2, 8.0],
    ],
)
@pytest.mark.parametrize(
    'divisor',
    [
        [2, -3, 8, -2, 3, 5],
        [2.1, -3.4, 8.0, -2.1, 3.4, 5.0],
    ],
)
@pytest.mark.parametrize('fmod', [0, 1])
```

Could you please add some [examples](https://github.com/onnx/onnx/blob/main/docs/Operators.md#examples-92) from ONNX Spec?